decorator.rb

```diff
@@ -0,0 +1,14 @@
+require './nameable'
+
```

Comment:
- [ ] [OPTIONAL] for such a small task it's better to create the classes in one file